### ISSUE TITLE
VAMC Policies: adopt V1 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16953]
## Description
VAMC Policies pages need the following updates to components:

Example pages:

- [Boston Policies](https://www.va.gov/boston-health-care/policies/)
- [Minneapolis Policies](https://www.va.gov/minneapolis-health-care/policies/)

### Elements that are not using components that should be (Backlog)
1. ~Link~
   - ~Links are not using a component but should be using [the v1 component](https://design.va.gov/components/link/) if possible, with the understanding that~ CMS content has limitations (Randi: confirming here that we can't change the way the CMS content is rendered)
2. ~Telephone~
   - ~[The component](https://design.va.gov/components/telephone) is not being used anywhere but should be if possible, with the understanding that CMS content has limitations~
   - ~Exts in phone numbers within the component are parsed correctly~
3. [Back to top component](https://design.va.gov/components/back-to-top)
   - Update 5/3: After determining that this component does not actually have built in intelligence to only show based on page height:
      -  This should still be added, the Policies pages are typically longer so having this on these pages will probably make sense more often than not. 

## Acceptance Criteria
- [ ] ~Link component has been added~
- [ ] Back to top component has been added
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

**Comment by maxx1128** [comment]
I estimate this ticket at a **2** for moving many relatively simple elements over to components (almost deja vu). I bumped it down from 3 since it's a simpler page with fewer risks of side effects from changing components.

**Comment by laflannery** [comment]
Telephone moved to #16148

**Comment by eselkin** [comment]
I imagine a 2, since all the links internal to the page are centralized content or drupal content that cannot be altered in the code currently.

**Comment by randimays** [comment]
@laflannery The body content on this page is Drupal content (WYSIWYG blocks), and we can't control the way the links render. I'll still add the Back to Top component on these and adjust the ACs accordingly.

**Comment by laflannery** [comment]
Thanks @randimays Thats what thought but just wanted to be sure I didn't miss anything. Thanks for confirming!
